README.md

```diff
@@ -445,11 +445,15 @@ matrix(
 Prettier ships with a handful of customizable format options, usable in both the CLI and API.
 
 ### Print Width
-Specify the length of line that the printer will wrap on.
-
-**We strongly recommend against using more than 80 columns.**
-
-Prettier works by cramming as much content as possible until it reaches the limit, which happens to work well for 80 columns but makes lines that are very crowded. When a bigger column count is used in styleguides, it usually means that code is allowed to go beyond 80 columns, but not to make every single line go there, like Prettier would do.
+Specify the line length that the printer will wrap on.
+
+> **For readability we recommend against using more than 80 characters:**
+>
+>In code styleguides the maximum amount of characters on a line can specified with a 'line length'. Line lengths are typically set to 120 characters.
+>
+>This is not the amount of characters that will be on a line. For readability, the average amount of characters will be 80 characters on a line.
```

Comment:
I still don't think this is particularly easy to understand, but I'm having trouble coming up with an alternative 🤷‍♂️